sdk/core/azure-core/test/ut/CMakeLists.txt

```
@@ -33,6 +33,11 @@ endif()
 
 include(GoogleTest)
 
+if (NOT WIN32)
+find_package(OpenSSL REQUIRED)
+SET(OPENSSLCRYPTO OpenSSL::Crypto)
```

nit: indent within an if-block
```suggestion
  find_package(OpenSSL REQUIRED)
  SET(OPENSSLCRYPTO OpenSSL::Crypto)
```